db/schema.rb

```diff
@@ -57,7 +57,7 @@
   end
 
   create_table "trips", force: :cascade do |t|
-    t.integer  "duration"
+    t.bigint   "duration"
```

Comment:
this makes sense. i agree that we might want to just make all zipcodes 5 digits though... seems neater. but great job spotting this one and getting trips load functional. 

http://s2.quickmeme.com/img/0f/0ff49113f737f6725161e579f345863f0a8b6d82d260aca7a3726c18b760b024.jpg